packages/neo-one-server-plugin-project/src/buildCommand.ts

```diff
@@ -113,15 +114,18 @@ export const buildCommand = ({ cli }: InteractiveCLIArgs) => {
     .option('--no-progress', "Don't output progress. Typically used for CI scenarios")
     .option('--reset', "Reset the local development environment of a NEO•ONE project to it's original state.")
     .option('--watch', 'Watch contracts for changes and rebuild.')
+    .option('--browser', 'Generated code references pre-bundled NEO•ONE client, ready to be used in the browser.')
```

Comment:
This needs to be an option in the `ProjectConfig` - https://github.com/neo-one-suite/neo-one/blob/master/packages/neo-one-server-plugin-project/src/types.ts#L36

Otherwise, for example, `neo-one build --reset` doesn't work as expected (see below).